src/test/skript/tests/syntaxes/functions/clamp.sk

```diff
@@ -41,3 +41,15 @@ test "clamp numbers":
 	assert number within {_got::1} is not number within {_got::1} with "(edge cases list) NaN" # need within because the variables weren't cooperating
 	assert {_got::2} is {_expected::2} with "(edge cases list) -infinity"
 	assert {_got::3} is {_expected::3} with "(edge cases list) infinity"
+
+test "clamp numbers (single)":
+	set {_expected::*} to (1, 0.0, and 2.0)
+	set {_got::*} to clamp((1, -infinity value, infinity value), 0.0, 2.0)
+	assert size of {_got::*} is 3 with "got %size of {_got::*}%"
```

Comment:
you know what i'm here about

Reply:
You're the grim reaper of my evening can you at least suggest the change so I don't have to open the IDE 😭

Reply:
☠️